Documentation/ExtensionArchitecture/DeclarationFile/Index.rst

```diff
@@ -34,7 +34,6 @@ values in the :php:`$EM_CONF` array if needed.
        'author_email' => 'john@example.org',
        'author_company' => 'some company',
        'state' => 'stable',
-       'createDirs' => '',
```

Comment:
Can you also drop this from *Deprecated Configuration* below?

Reply:
The information about deprecated or removed configuration shall be kept there. Many extensions will support older versions of TYPO3 as well.

Reply:
But this does not reflect the current workflow. Each doc main version only refers to its latest state and the changes to the previous version. Otherwise we end up with a huge list of outdated things in the latest docs after a few years.

So we should keep the current wording and really drop everything which was dropped with the latest version(s). People can always switch to the docs for older versions if needed.

Reply:
But then I must remove all of them. These are not used any more in TYPO3 10:

        // Remove TER v1-style entries
        unset($emConf['dependencies']);
        unset($emConf['conflicts']);
        unset($emConf['suggests']);
        unset($emConf['private']);
        unset($emConf['download_password']);
        unset($emConf['TYPO3_version']);
        unset($emConf['PHP_version']);
        unset($emConf['internal']);
        unset($emConf['module']);
        unset($emConf['loadOrder']);
        unset($emConf['lockType']);
        unset($emConf['createDirs']);
        unset($emConf['shy']);
        unset($emConf['priority']);
        unset($emConf['modify_tables']);
        unset($emConf['CGLcompliance']);
        unset($emConf['CGLcompliance_note']);

Reply:
Great, even better. :-)

Reply:
I have removed everything below deprecated. All of them have been deprecated in former versions of TYPO3.

Reply:
... with a hint to switch the page version, thanks!